### DISCUSSION TITLE
Placeholder on Datetime field

### DISCUSSION THREAD

**Discussion by mickeymonz** [directus/directus#8411] [Feature Requests]
I would like to custom placeholder on Datetime field instead "Enter a value ..."

![Screen Shot 2021-09-29 at 1 10 53 PM](https://user-images.githubusercontent.com/63136554/135212696-46dec12f-4527-4136-8393-286f3ace0c72.png)

**Comment by azrikahar** [comment]
Might be somewhat related to the discussion/exchange such as this: https://github.com/directus/directus/pull/8104#issuecomment-921720953. Not really a concrete solution/response per se, but just wanted to give you a brief acknowledgement on this 👍

**Comment by benhaynes** [comment]
Several of our core interfaces have options to change placeholder text, no reason this one shouldn't too. We'd be happy to review a PR!

**Comment by br41nslug** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
